Whiskey/Whiskey.psd1

```diff
@@ -163,6 +163,7 @@
             ReleaseNotes = @'
 * Added `InstallNode` task for installing Node.js.
 * Fixed: the ProGetUniversalPackage task ignores and overwrites release and build metadata in the `Version` property.
+* Whiskey now uses global PowerShell modules rather than always installing a private copy in a repository's PSModules folder. Private modules in "PSModules" are always used instead of global modules.

```

Comment:
Should specifically call out that this changes the behavior of the `GetPowerShellModule` task (in a breaking way?). That task will no longer download the given PowerShell module if it's available globally on the system via the `$PSModulePath` environment variable.

Reply:
It still does. When given a `Path`, it will install the module if it is not at that path. See https://github.com/webmd-health-services/Whiskey/pull/352/files?file-filters%5B%5D=.ps1&file-filters%5B%5D=.psd1&file-filters%5B%5D=.yml&hide-deleted-files=true#diff-e86b4c8e4aea7c2ab2e71bfeef1f1d86c4f55e9295b683fae6a089e4c6ff8f93R99-R101 .